Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/csi-addons/controller:latest
```

Comment:
Maybe this should include `kubernetes` or `k8s` in the name? The controller is Kubernetes specific and will not work on other container platforms.